docs/reference/changelog.rst

```diff
@@ -21,6 +21,10 @@ Changelog
 2.25.0 (unreleased)
 -------------------
 
+New features:
+
+- Enforce filesystem mounts to be ordered in increasing ``mount`` depth.
```

Comment:
```suggestion
- Order filesystem mounts by increasing ``mount`` depth.
```

Just a nitpick, not required.

Reply:
How about something more spelled out, like "The LifecycleManager will now validate that filesystem mounts are ordered in increasing 'mount' depth"?

Reply:
It looks better, thanks! But I realized that the "ordered by `depth`" is an oversimplification because the following would be valid:
- /
- /a
- /a/foo
- /b
- /b/bar

`/a/foo` is deeper than `/b` but is not conflicting, and the code properly deal with that.

So I will rework this sentence to clarify that.